packages/assets-controllers/src/NftController.test.ts

```diff
@@ -806,9 +924,9 @@ describe('NftController', () => {
             interactingAddress: OWNER_ADDRESS,
             asset: {
               ...ERC721_NFT,
-              description: null,
-              image: null,
-              name: null,
+              description: 'testERC721Description',
+              image: 'testERC721Image',
+              name: 'testERC721Name',
```

Comment:
The result for this test changed with all messenger calls being fully mocked. Is this an error?